src/constants/versions.ts

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  ANDROID_VERSION: "1.28.1",
+  KOTLIN_VERSION: "0.12.1",
```

Comment:
```suggestion
  ANDROID_VERSION: "1.28.3",
  KOTLIN_VERSION: "0.12.3",
```

Reply:
Maybe `ANDROID_KOTLIN_VERSION` is a better name? `KOTLIN_VERSION` sounds like the version of Kotlin language.